networkx/algorithms/centrality/tests/test_closeness_centrality.py

```diff
@@ -7,30 +7,16 @@
 import networkx as nx
 
 
-class TestClosenessCentrality:
-    @classmethod
-    def setup_class(cls):
-        cls.K = nx.krackhardt_kite_graph()
-        cls.P3 = nx.path_graph(3)
-        cls.P4 = nx.path_graph(4)
-        cls.K5 = nx.complete_graph(5)
-
-        cls.C4 = nx.cycle_graph(4)
-        cls.T = nx.balanced_tree(r=2, h=2)
-        cls.Gb = nx.Graph()
-        cls.Gb.add_edges_from([(0, 1), (0, 2), (1, 3), (2, 3), (2, 4), (4, 5), (3, 5)])
-
-        F = nx.florentine_families_graph()
-        cls.F = F
+@pytest.fixture()
+def undirected_G():
+    G = nx.fast_gnp_random_graph(n=100, p=0.6, seed=123)
+    cc = nx.closeness_centrality(G)
+    return G, cc
 
-        cls.LM = nx.les_miserables_graph()
-
-        # Create random undirected, unweighted graph for testing incremental version
-        cls.undirected_G = nx.fast_gnp_random_graph(n=100, p=0.6, seed=123)
-        cls.undirected_G_cc = nx.closeness_centrality(cls.undirected_G)
 
+class TestClosenessCentrality:
     def test_wf_improved(self):
-        G = nx.union(self.P4, nx.path_graph([4, 5, 6]))
+        G = nx.union(nx.path_graph(4), nx.path_graph([4, 5, 6]))
```

Comment:
Is there ever a possibility of overlapping node labels?

Reply:
Looks like this test was designed not to have overlapping node. Result has `0,1,2,3,4,5,6`.

But the issue of overlapping nodes is a issue/test for the `union` function. Here we are testing `wf`

Reply:
LGTM